spec/warden/jwt_auth/env_helper_spec.rb

```diff
@@ -23,15 +23,25 @@
     it 'returns REQUEST_METHOD' do
       env = { 'REQUEST_METHOD' => 'POST' }
 
-      expect(described_class.request_method(env)) == 'POST'
+      expect(described_class.request_method(env)).to eq('POST')
     end
   end
 
   describe '::authorization_header(env)' do
-    it 'returns HTTP_AUTHORIZATION' do
+    it 'returns HTTP_AUTHORIZATION by default' do
       env = { 'HTTP_AUTHORIZATION' => 'Bearer 123' }
 
-      expect(described_class.authorization_header(env)) == 'Bearer 123'
+      expect(described_class.authorization_header(env)).to eq('Bearer 123')
+    end
```

Comment:
We can remove this test altogether now. This way is consistent with what we're doing now with `aud_header`